honeybee_vtk/camera.py

```diff
@@ -0,0 +1,246 @@
+        # Horizontal view angle
+        camera.SetViewAngle(self._h_size.value)
+        camera.SetUseHorizontalViewAngle(True)
+        camera.UseHorizontalViewAngleOn()
+
+        return camera
+
+    def _adjusted_position(self):
+        """Get adjusted camera position.
+
+        This method helps bring camera close to the model within an offset distance.
+
+        Returns:
+            Adjusted camera position in the form of a tuple with three decimal values.
+        """
+        index = self._flat_view_directions[self._direction.value][0]
+        nearest_point = self._outermost_point()
+        cord = nearest_point[index]
+
+        # If the cordinate we're looking for is negative the offset needs to be applied
+        # in negative as well
+        if cord <= 0:
+            offset = self._camera_offset * -1
+        else:
+            offset = self._camera_offset
+
+        # if the camera needs to move along x-axis
+        if index == 0:
+            adjusted_position = (cord + offset, self._position.value[1],
+                                 self._position.value[2])
+
+        # if the camera needs to move along y-axis
+        elif index == 1:
+            adjusted_position = (self._position.value[0], cord + offset,
+                                 self._position.value[2])
+
+        # if the camera needs to move along z-axis
+        elif index == 2:
+            adjusted_position = (self._position.value[0], self._position.value[1],
+                                 cord + offset)
+
+        return adjusted_position
+
+    def _outermost_point(self):
+        """Find the outermost point in a Model.
+
+        This method looks at the bounds of the actors and finds the outermost point
+        in actors from a certain direction. For example, if you are looking at the model
+        from top and you wished to know the outermost point in +Z, this method helps you
+        find that point. This point is used in creating adjusted camera position when
+        parallel projection is requested.
+
+        Returns:
+            A Ladybug Point3D object.
+        """
+        # Check axis(index) and direction to find the nearest point
+        index, dir = self._flat_view_directions[self._direction.value]
+
+        # dictionary with cordinate(int):Point3D structure
+        cord_point = {point[index]: point for point in self._bounds}
+
+        # if z-axis
+        if index == 2:
+            if dir == '+':
+                outermost_point = cord_point[sorted(cord_point, reverse=True)[0]]
+            else:
+                outermost_point = cord_point[sorted(cord_point)[0]]
+        # if x-axis or y-axis
+        else:
+            if dir == '+':
+                outermost_point = cord_point[sorted(cord_point)[0]]
+            else:
+                outermost_point = cord_point[sorted(cord_point, reverse=True)[0]]
+
+        return outermost_point
+
+
+    @classmethod
+    def from_model(cls, model):
```

Comment:
@mostaphaRoudsari, do you mean that this method should be called in the Model's constructor and we should have a list of cameras as property of a Model object rather than views?

Reply:
Yes. I think it's fine for a VTK Model to have `camera`s and not `view`s.

Reply:
Compplied.